app/Models/Member.php

```diff
@@ -14,8 +14,9 @@ class Member extends Authenticatable
     const STATUS_CLOSE = 0;
     const GENDER_MALE = 0;
     const GENDER_FEMALE = 1;
+    const ID = 0;
```

Comment:
a thấy e toàn truyền ID = 0 sang view để làm gì nhỉ

Reply:
Em dùng để tăng giá trị cho order id trong bang anh ạ.